crates/bevy_pbr/src/meshlet/dummy_visibility_buffer_resolve.wgsl

```diff
@@ -0,0 +1,9 @@
+#define_import_path bevy_pbr::meshlet_visibility_buffer_resolve
+
+struct VertexOutput {
+    dummy: u32
+}
+
+fn resolve_vertex_output(frag_coord: vec4<f32>) -> VertexOutput {
+    return VertexOutput(1717u);
```

Comment:
I need to write docs, but this is basically a dummy shader for when the MeshletPlugin is not loaded, so that naga_oil doesn't complain about missing imports :/. Shaderdefs is not enough to fix this unfortunately.